### ISSUE TITLE
[Accessibility] [Mini Cart] The kebab menu items are in the focus order when they are not visible

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-cif-components#692]
**Steps to Reproduce**
1. Open page in Google Chrome.
2. Use TAB or TAB + SHIFT to navigate through kebab menus in the cart components.
3. Notice the "disappearance" of focus as it moves through the visually hidden menu items.

**Current Behavior**
The kebab menu options receive keyboard focus even when the menu is not expanded. Sighted keyboard users cannot tell which element has focus.

**Expected behavior/code**
Visually-hidden elements should not receive keyboard focus. 

**Environment**
Chrome 92, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Sighted keyboard-only users

**WCAG SC**
2.4.3 Focus Order

**Possible Solution**
Consider using the WAI-ARIA Authoring Practices [Menu Button](https://www.w3.org/TR/wai-aria-practices-1.1/#menubutton) design pattern , implementing all expected roles, states, properties, and keyboard interactions.

**Comment by LSantha** [comment]
Deprecated component.
